notebook_index.yml

```diff
@@ -169,6 +170,11 @@ contents:
                 notebooks:
                   - supercritical_steam_cycle: Simulate a typical supercritical steam cycle
                   - supercritical_power_plant: Simulate a typical supercritical powerplant including boiler and steam cycle
+          -
+            name: temperature_swing_adsorption
+            title: Temperature Swing Adsorption Using 1D Fixed Bed Model
```

Comment:
Should the title be in quotation marks?  (`title: "Temperature Swing Adsorption Using 1D Fixed Bed Model"`)